about/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="about">About</string>
+    <string name="privacy_policy">Privacy policy</string>
+    <string name="version">Version %1$s</string>
+    <string name="copy">Copy</string>
+    <string name="oops_something_went_wrong">Oops! Something went wrong</string>
+    <string name="about_download_qr_image">Download QR Image</string>
+    <string name="about_wallet_copied">Wallet copied!</string>
+    <string name="ark_support_email">support@ark-builders.dev</string>
+    <string name="about_send_email_part_1">"Send us the transaction id by email "</string>
+    <string name="about_send_email_part_2">" to receive premium support. We’ll make our best to help you with any issue you encounter while using our Software."</string>
```

Comment:
These strings have starting/ending double quotes.
Is that something required for `AnnotatedString` to work?

Reply:
Double quotes are needed for leading and trailing spaces